lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
-    color: $base-font-color;
-  }
-
-  .cart-table {
-    width: 100%;
-
-    tr:nth-child(even) {
-      background-color: $gray-l6;
-      border-bottom: 1px solid $gray-l5;
-    }
-
-    tr.always-gray{
-      background: #eee !important;
-      border-top: 2px solid $white;
-    }
-
-    tr.always-white {
-      background: $white !important;
-
-      td {
-        padding: ($baseline*1.5) 0 ($baseline/2);
-      }
-    }
-
-    tr td.cart-total {
-      padding: ($baseline/2) 0;
-
-      span {
-        display: inline-block;
-        margin-right: ($baseline*0.75);
-        margin-left: ($baseline*0.75);
-        font-weight: bold;
-      }
-    }
-
-    .cart-headings {
-      height: 35px;
-      border-bottom: 1px solid #BEBEBE;
-
-      th {
-        text-align: left;
-        border-bottom: 1px solid $border-color-1;
-
-        &.qty {
-          width: 100px;
-        }
-
-        &.u-pr {
-          width: 70px;
-        }
-
-        &.prc {
-          width: 150px;
-        }
-
-        &.cur {
-          width: 100px;
-          text-align: center;
-        }
-
-        &.dsc{
-          width: 640px;
-          padding-right: 50px;
-          text-align: left;
-        }
-      }
-    }
-
-    .cart-items {
-      td {
-        padding: ($baseline/2) 0;
-        position: relative;
-        line-height: normal;
-
-        span.old-price {
-          position: relative;
-          text-decoration: line-through;
-          color: red;
-          font-size: 12px;
-          top: -1px;
-          margin-left: 3px;
-        }
-      }
-
-      td:nth-child(3) {
-        text-align: center;
-      }
-
-      td:last-child {
-        width: 50px;
-        text-align: center;
-      }
-
-      td:nth-child(1) {
-        line-height: 22px;
-        text-align: left;
-        padding-right: $baseline;
-      }
-    }
-
-    .cart-totals {
-      td {
-        &.cart-total-cost {
-          font-size: 1.25em;
-          font-weight: bold;
-          padding: ($baseline/2) 25px;
-        }
-      }
-    }
-  }
-
-  table.order-receipt {
-    width: 100%;
-
-    .order-number {
-      font-weight: bold;
-    }
-    .order-date {
-      text-align: right;
-    }
-    .items-ordered {
-      padding-top: ($baseline*2.5);
-    }
-
-    tr {
-    }
-
-    th {
-      text-align: left;
-      padding: 25px 0 15px 0;
-
-      &.qty {
-        width: 50px;
-      }
-
-      &.u-pr {
-        width: 100px;
-      }
-
-      &.pri {
-        width: 125px;
-      }
-
-      &.curr {
-        width: 75px;
-      }
-    }
-
-    tr.order-item {
-      td {
-        padding-bottom: ($baseline/2);
-
-        span.old-price {
-          text-decoration: line-through !important;
-        }
-      }
-    }
-  }
+  margin-bottom: $baseline;
+  border-bottom: 1px solid $pink;
+  padding: ($baseline/2) 0;
+  background: $card-errors-background-color;
+  color: $pink;
+  font-family: $sans-serif;
 }
 
 #expiry-msg {
-  padding: 15px;
+  @extend %t-icon6;
```

Comment:
awesome! thanks for updating these to be consistent with the Concentric CSS property ordering! (https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style.css) :100: